src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -159,11 +159,7 @@ public class HubEndPoint<THub, TClient> : EndPoint, IInvocationBinder where THub
                 {
                     while (connection.Transport.Input.TryRead(out var incomingMessage))
                     {
-                        if (!protocol.TryParseMessage(incomingMessage.Payload, this, out var hubMessage))
-                        {
-                            _logger.LogError("Received invalid message");
-                            throw new InvalidOperationException("Received invalid message");
-                        }
+                        var hubMessage = protocol.ParseMessage(incomingMessage.Payload, this);
```

Comment:
Handle null values returned by ParseMessage (+test if possible) - https://github.com/aspnet/SignalR/pull/390/commits/fa7ead1f51ac45c4cc928d6725e43cedc45d66b9#diff-17d494f97b6bf3a80750f65894639dccR77 (or throw at the source?)